### ISSUE TITLE
Make kneeboard flight names match custom flight names

### ISSUE THREAD

**Issue by funkyschnitzel** [dcs-liberation/dcs_liberation#862]
As per the title, I would love if the kneeboard flight names were the same as the custom flight names (where custom names have been added).
For the purposes of running larger multiplayer games it's important that everyone knows which flight they are on, and having different flight names on the kneeboard adds confusion.
On a similar vein, would it be possible (where no custom name has been assigned) for the multiplayer role selection screen to show the flight name that will be on the kneeboard, rather than simply the aircraft type and mission? This will help people to get into the correct aircraft at mission start.

**Comment by benedikt-wegmann** [comment]
The latter ist probably a limitation of DCS.
Regarding the kneeboard I want to understand anyway how to include more information on it (headings and more frequencies), I will attempt to make this change as well.

**Comment by DanAlbert** [comment]
@benedikt-wegmann https://github.com/Khopa/dcs_liberation/blob/develop/gen/kneeboard.py is a good place to start for anything kneeboard related. lmk if I can help.

**Comment by benedikt-wegmann** [comment]
@DanAlbert I'll start to sniff around there, thanks for the pointer.

**Comment by DanAlbert** [comment]
A quirk to be aware of is that the kneeboard is generated using [FlightData](https://github.com/Khopa/dcs_liberation/blob/8e51b7fc1d745cec392895716c30d4ddcf7beb78/gen/aircraft.py#L260) rather than the normal [Flight]() class, since the latter doesn't include all the stuff that gets generated by the mission like radio frequencies and whatnot.

That gets created [here](https://github.com/Khopa/dcs_liberation/blob/develop/gen/aircraft.py#L787). Can either add the new bit of data for the flight name, or a reference to the `Flight`.

**Comment by benedikt-wegmann** [comment]
@DanAlbert Thanks für pointing me to `FlightData`, probably still wouldn't have found it. :joy:

**Comment by benedikt-wegmann** [comment]
@funkyschnitzel I just noticed that the custom name becomes part of the *group name* in the flight selection screen. My recommendation for your scenario is to give each human flight a custom name identifying the player in some way (nickname etc.). In the flight selection screen when joining the server each player can now identify their flights easily. Once in the cockpit they just need to flip through the knee board pages to find the page that has their matching name in the title (with this patch), this will be their briefing with call sign, frequencies and so forth.

**Comment by DanAlbert** [comment]
I don't think there's a need to rename the pilot as well; group should be enough?

**Comment by benedikt-wegmann** [comment]
I didn't explain it very well. I only referred to the custom name of the flight, not the pilot. But if one were to name the flight "Bob's flight" or "Strike 1" or similar, something that the other player can identify as his flight, than he will be able to pick the correct flight from the flight selection screen when joining the mission because the custom flight name becomes part of the group name in said list I noticed. This should satisfy this issue.
